### ISSUE TITLE
Slider Accessibility: Does it announce "disabled" or Checked/Unchecked State

### ISSUE THREAD

**Issue by amarlette** [callstack/react-native-slider#265]
# Slider Accessibility
In regaurds to facebook/react-native#30939 & facebook/react-native#30961 has react-native-slider already addressed these accessibility issues? Any help in identifying if this issue has been addressed in this repo would be greatly appreciated!

### Expected results
When a component is "disabled", screen readers should announce this as part of their focus announcement. Clicks to these elements should also have no effect.

### Expected results
Upon focus of an element with the "checked" accessibilityState, the screen reader should announce either "checked" or "not checked", or in the case of elements with the "switch" role, it should announce "on" or "off". Upon activation, if the state is changed, the new state should be announced.

**Comment by BartoszKlonowski** [comment]
Hello @amarlette!
I have a question regarding the second case:
**Slider Component does not announce Checked/Unchecked State**:

As I can see the [AccessibilityState.checked](https://reactnative.dev/docs/accessibility#accessibilitystate) should apply to binary-states components like switches, checkboxes, etc.
`<Slider/>` on the other hand may have many values, so it cannot be turned on/off. It can be disabled, but it falls down to the category of the first case you've mentioned.

Could you elaborate on this? How would `<Slider/>` apply to `AccessibilityState={checked: true}`?
